graylog2-server/src/main/java/org/graylog2/indexer/Constants.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog2.indexer;
+
+class Constants {
+    static final String ES_DATE_FORMAT = "yyyy-MM-dd HH:mm:ss.SSS";
```

Comment:
According to the [breaking changes docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking-changes-7.0.html#_joda_based_date_formatters_are_replaced_with_java_ones) for ES 7, we can use the `8` prefix for date patterns in ES 6. (supported since 6.7) That way users don't get warnings if they create new indices with ES 6.

We requite at least ES 6.8 to run Graylog now, correct?

Reply:
Correct!